src/codewhisperer/service/transformationHubViewProvider.ts

```diff
@@ -185,11 +185,27 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
             }
         }
         const isJobInProgress = transformByQState.isRunning()
+        const codiconsUri = this._view?.webview.asWebviewUri(
+            vscode.Uri.joinPath(this._extensionUri, 'node_modules', '@vscode/codicons', 'dist', 'codicon.css')
```

Comment:
This looks strange. No other webview needs to reach into `node_modules`, why is it needed here? Are there not existing codicon usage examples that can be found in existing webviews in this codebase?

Reply:
I previously tried https://github.com/aws/aws-toolkit-vscode/blob/f355be267fc59a116a9f697c0163f4f2e8c5e45b/src/codewhisperer/views/securityPanelViewProvider.ts#L52 and also the `getIcon()` used in many other places, neither of which worked, so I took this approach, but regardless the latest work accomplishes the same in a simpler way. I didn't test with the VSIX but I imagine it would fail since `node_modules` is not there; good to know.